main_routers/system_router.py

```
@@ -36,6 +37,126 @@
+            for t in (twitter.get('trending', []) or [])[:3]:
+                title = t.get('name', '') or t.get('word', '')
+                if title and t.get('url'):
+                    links.append({'title': title, 'url': t['url'], 'source': 'Twitter'})
+    except Exception as e:
+        logger.warning(f"提取链接失败 [{mode}]: {e}")
+    return links
+
+
+def _parse_web_screening_result(text: str) -> dict | None:
+    """
+    解析 Phase 1 Web 筛选 LLM 的结构化结果，提取链接信息。
```

Ruff (0.15.1)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)